packages/env/README.md

```diff
@@ -175,6 +175,30 @@ Positionals:
             [string] [choices: "all", "development", "tests"] [default: "tests"]
 ```
 
+### `wp-env run [container] [command]` 
+
+```sh
+wp-env run <container> [command..]
+
+Runs an arbitrary command in one of the underlying Docker containers.
+
+Positionals:
```

Comment:
What does "positional" mean?

Reply:
Positional argument, it's following the same convention as the part where `wp-env clean [environment]` is documented